src/libstd/io/error.rs

```diff
@@ -233,8 +232,7 @@ impl Error {
     ///
     /// If this `Error` was constructed via `new` then this function will
     /// return `Some`, otherwise it will return `None`.
```

Comment:
We should add a note to the docs for these methods about the need to call via UFCS - it confused the hell out of me for a while when I was writing tests for them.